packages/swirl-components/src/components/swirl-modal/swirl-modal.css

```diff
@@ -283,13 +283,13 @@
   display: flex;
   overflow: hidden;
   width: 100vw;
-  max-width: 40rem;
   height: var(--swirl-modal-view-height);
   background-color: var(--s-surface-overlay-default);
   flex-direction: column;
 
   @media (--from-tablet) {
     width: 90vw;
+    max-width: 40rem;
     max-height: var(--swirl-modal-max-height);
```

Comment:
**suggestion (non-blocking):** maybe we can intoduce a max-width property similar to the max-height? :)